XamlControlsGallery/ControlPages/GridViewPage.xaml

```diff
@@ -114,9 +289,11 @@
                             Tag="TextTemplate" />
                         <ToggleButton Margin="0,8,0,0" Click="FlowDirectionCheckBox_Click" Content="Reverse FlowDirection" />
                         <TextBlock Margin="0,18,0,10" Text="GridView Properties" />
+                        <TextBlock TextWrapping="Wrap" FontSize="13" MaxWidth="150"> In order to drag, drop, and reorder items within the GridView, make sure all boxes are checked below.</TextBlock>
```

Comment:
Interestingly, I was able to reorder the items without having to check the "IsItemClickEnabled" checkbox. This was also possible when selection mode was set to none. So this text is a bit misleading I think.

Reply:
Tried to clarify a bit in my upcoming commit. Thanks for pointing this out!